dali/image/tiff.cc

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) 2017-2018, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/image/tiff.h"
+
+namespace dali {
+
+namespace {
+
+const std::vector<int> header_intel = {77, 77, 0, 42};
```

Comment:
If this collection is meant to be constant, you might want to declare it as an std::array. Moreover, since it has ints which are constexpr constructible you can give it value at compile time by declaring it constexpr std::array

Reply:
That's good point, I'll do it